students/vvinodh/Lesson1/test_unit.py

```diff
@@ -0,0 +1,113 @@
+"""Test File for testing inventory management files"""
+import sys
+sys.path.append('/UW-Python-Advanced/SP_Python220B_2019/students/vvinodh/Lesson1/inventory_management')
```

Comment:
Hi Vinod,

Looks like you're following Unix-style format, but if I remember correctly, you're on a Windows system. You'll need to use a full path, for example:

`sys.path.append(r'C:\UW-Python-Advanced\SP_Python220B_2019\students\vvinodh\Lesson1\inventory_management')`

Note that in this example, I'm assuming your _UW-Python-Advanced_ folder is in the root directory.

Regards,

Luis

Reply:
PS C:\UW-Python-Advanced\SP_Python220B_2019\students\vvinodh\Lesson1> python -m unittest test_unit.py
Traceback (most recent call last):
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\unittest\__main__.py", line 18, in <module>
    main(module=None)
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\unittest\main.py", line 94, in __init__
    self.parseArgs(argv)
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\unittest\main.py", line 141, in parseArgs
    self.createTests()
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\unittest\main.py", line 148, in createTests
    self.module)
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\unittest\loader.py", line 219, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\unittest\loader.py", line 219, in <listcomp>
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "C:\Users\vinodh\AppData\Local\Programs\Python\Python36\lib\unittest\loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "C:\UW-Python-Advanced\SP_Python220B_2019\students\vvinodh\Lesson1\test_unit.py", line 3
    sys.path.append('C:\UW-Python-Advanced\SP_Python220B_2019\students\vvinodh\Lesson1\inventory_management')
                   ^